purchase_work_acceptance_evaluation/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
By concept

Configuration data (both model and menu)
- Only purchase_manager, can CRUD (create, read, update, delete), 
- purchase_user can readonly.

Transaction data
- All user can CRUD (leave group = <blank>